packages/url_launcher/url_launcher_android/CHANGELOG.md

```diff
@@ -1,7 +1,3 @@
-## 6.0.33
```

Comment:
A straight revert won't work for a published plugin; you have to increase the version and add a new entry.

Reply:
manually edited to return 33 changelog and add a 34 changelog, Manual edit to point pubspec to 34.